packages/client/components/Avatar/Avatar.tsx

```diff
@@ -1,19 +1,28 @@
-import React, {forwardRef} from 'react'
-import AvatarBadge from '../AvatarBadge/AvatarBadge'
 import styled from '@emotion/styled'
+import React, {forwardRef} from 'react'
 import defaultUserAvatar from '../../styles/theme/images/avatar-user.svg'
+import AvatarBadge from '../AvatarBadge/AvatarBadge'
 
-type ImageBlockProps = Pick<Props, 'sansRadius' | 'sansShadow' | 'picture' | 'size' | 'onClick'>
+type ImageBlockProps = Pick<
+  Props,
+  'sansRadius' | 'sansShadow' | 'picture' | 'size' | 'onClick' | 'isConnected'
+>
 
 const ImageBlock = styled('div')<ImageBlockProps>(
-  ({sansRadius, sansShadow, picture, size, onClick}) => ({
-    backgroundImage: `url(${picture}), url(${defaultUserAvatar})`,
+  ({isConnected, sansRadius, sansShadow, picture, size, onClick}) => ({
+    backgroundImage: `${
+      isConnected === false ? 'linear-gradient(rgba(255,255,255,.65), rgba(255,255,255,.65)),' : ''
+    } url(${picture}), url(${defaultUserAvatar})`,
+    ':hover': {
+      backgroundImage: `linear-gradient(rgba(255,255,255,.5), rgba(255,255,255,.5)),
+    url(${picture}), url(${defaultUserAvatar})`
+    },
     backgroundPosition: 'center center',
     backgroundRepeat: 'no-repeat',
     backgroundSize: 'cover',
     borderRadius: sansRadius ? 0 : '100%',
     boxShadow: sansShadow ? 'none' : undefined,
-    cursor: onClick ? 'pointer' : 'default',
+    ...(onClick ? {cursor: 'pointer'} : null),
```

Comment:
+1 why set the cursor to null instead of default? I'd prefer to following the existing pattern of `cursor: onClick ? 'pointer' : 'default'`

Reply:
I just thought that because `cursor` is an inherited CSS property, we might not want to override the value if the avatar doesn't have a click handler...

Reply:
But I agree that it's may be now is not the right time to change cursor pattern, so I'm going to revert this change.